app/mm-redux/selectors/entities/groups.ts

```diff
@@ -63,30 +112,53 @@ export const getGroupsNotAssociatedToTeam = reselect.createSelector(
     getAllGroups,
     (state: GlobalState, teamID: string) => getTeamGroupIDSet(state, teamID),
     (allGroups, teamGroupIDSet) => {
-        return Object.entries(allGroups).filter(([groupID]) => !teamGroupIDSet.has(groupID)).map((entry) => entry[1]);
+        return Object.values(allGroups).filter((group) => !teamGroupIDSet.has(group.id));
```

Comment:
can `allGroups` be null or undefined? if it can be this would crash. Same as with the others....